poe-tasks/poetry-connector-tasks.toml

```diff
@@ -135,6 +135,41 @@ help = "Get the language of the connector from its metadata.yaml file. Use with
 cmd = """yq eval '.data.connectorBuildOptions.baseImage' $PWD/metadata.yaml"""
 help = "Get the base image of the connector from its metadata.yaml file."
 
+[tool.poe.tasks.get-local-version]
+cmd = """yq eval '.data.dockerImageTag' $PWD/metadata.yaml"""
+help = "Get the version of the connector from its metadata.yaml file. Use with -qq to get just the version."
+
+[tool.poe.tasks.create-connector-registry-artifacts]
+shell = '''
+set -eu
+VERSION=$(poe -qq get-local-version)
+CONNECTOR_NAME=$(poe -qq get-connector-name)
+
+# Create build directory
+mkdir -p "build/releases/${VERSION}"
+
+# Clear existing contents
+rm -rf "build/releases/${VERSION}"/*
+
+# Copy required files
+REQUIRED_FILES=("metadata.yaml" "icon.svg")
```

Comment:
For all of these scripts, let's move the array declarations for required and optional files to the top of the script, and ideally formatted so each entry is on a different line (reduces reformatting and touching other lines when items are added and removed).